src/Tribe/PUE/Checker.php

```diff
@@ -493,7 +493,7 @@ private function get_api_message( $info ) {
 		 */
 		public function display_license_error_message() {
 			$plugin_info  = $this->json_error;
-			$license_tab = admin_url( 'wp-admin/edit.php?post_type=tribe_events&page=tribe-common' );
+			$license_tab = admin_url( 'edit.php?page=tribe-common&tab=licenses&post_type=tribe_events' );
```

Comment:
Good point, I'm using that url to replace this text: "Please go to the Events > Settings > Licenses tab to enter your license key." which would also be irrelevant in this case (I think).  Suggestions?